modules/ROOT/pages/tgw-manage-arm.adoc

```diff
@@ -0,0 +1,101 @@
+= Manage Transit Gateway Attachments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
+
+
+Use Runtime Manager to manage transit gateway attachments.
+
+== Rename a Transit Gateway Attachment
+
+To rename a transit gateway attachment:
+
+. On the *Transit Gateways* page, click the Transit Gateway menu (*...*) and select *Edit name*:
++
+_update screenshot - content and styles changed_
++
+image::tgw-rename.png[Edit name option in the Transit Gateways menu]
+. In the popup window, enter a new name and click *Save*.
+
+
+== Remove a Route from the Transit Gateway Route Table
+
+To remove a route:
+
+. On the *Transit Gateways* page, in the *VPCs* section, click the *X* icon in the row for a TGW route in the route table:
++
+_update screenshot - content and styles changed_
++
+image::tgw-delete-route.png[Remove route icon in the VPCs section]
+
+. At the confirmation prompt, click *Remove*.
+
+== Detach an Anypoint VPC from the Transit Gateway
+
+Because detaching a VPC from a transit gateway also deletes the attachment in AWS,
+undeploy any apps deployed to the VPC before detaching the VPC.
```

Comment:
I don't think this is appropriate guidance, because detaching doesn't affect the VPC itself. It just means apps deployed to the VPC won't be able to reach anything through the TGW anymore. The user will probably still want to keep their apps running.